get_helm.sh

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env bash
```

Comment:
Should this script be placed somewhere better than the repo root?

Reply:
sounds good, I should probably run from the service-bus directory